examples/Overlay.main.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Overlay } from '@itwin/itwinui-react';
+
+const wrapperStyle = {
+  border: '1px solid var(--iui-color-border)',
+  position: 'relative',
+  marginBottom: '12px',
+  maxWidth: '600px',
+  maxHeight: '400px',
```

Comment:
some of these styles (specifically `marginBottom` and `position`) are unnecessary

same comment in `Overlay.sub.tsx`

Reply:
I removed `marginBottom`, `maxWidth`, and `maxHeight`. `position` is there to keep the Overlay from being applied over the entire LiveExample viewport (it was covering the codesandbox icon in the bottom left).

Reply:
but OverlayWrapper already has `position: relative` in the css

Reply:
what could I put to keep the wrapper from overtaking the viewport?

Reply:
nothing. OverlayWrapper already has `position: relative`, so it should look fine with just this code:

```jsx
import * as React from 'react';
import { Overlay } from '@itwin/itwinui-react';

export default () => {
  return (
    <Overlay
      content={'loading...'}
      style={{ border: '1px solid var(--iui-color-border)' }}
    >
      <div>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
        mollit anim id est laborum.
      </div>
    </Overlay>
  );
};

```

Reply:
![image](https://github.com/iTwin/iTwinUI/assets/105806085/57b0718b-8918-467f-a8e4-70a1dc68fc69)

this is what it looks like, the `Overlay` takes over the entire LiveExample. I was asking to find a way to confine it to just the sample text inside the `Overlay` component.

Reply:
For some reason you've reverted the other change to `OverlayWrapper`. https://github.com/iTwin/iTwinUI/pull/1328#discussion_r1242385470

If you change it back, it works correctly.
![](https://github.com/iTwin/iTwinUI/assets/9084735/588aceca-774e-458e-b344-bf942f8c933e)
![](https://github.com/iTwin/iTwinUI/assets/9084735/bfcca1af-41f6-43f4-9ecc-93c01dfa45af)

Reply:
Ok, got it working correctly, thanks